packages/flutter/lib/src/services/system_channels.dart

```diff
@@ -26,16 +26,19 @@ class SystemChannels {
   ///  * `pushRoute`, which is called with a single string argument when the
   ///    operating system instructs the application to open a particular page.
   ///
+  ///  * `pushRouteInformation`, which is called with a map, which contains a
+  ///    location string, a state object and a restoration config, when the
+  ///    operating system instructs the application to open a particular page.
+  ///    These parameters are store under the key`location`, `state` and
+  ///    `restorationData` in the map.
+  ///
   /// The following methods are used for the opposite direction data flow. The
   /// framework notifies the engine about the route changes.
   ///
-  ///  * `routePushed`, which is called when a route is pushed. (e.g. A modal
-  ///    replaces the entire screen.)
-  ///
-  ///  * `routePopped`, which is called when a route is popped. (e.g. A dialog,
-  ///    such as time picker is closed.)
+  ///  * `routeUpdated`, which is called when current route has changed.
   ///
-  ///  * `routeReplaced`, which is called when a route is replaced.
```

Comment:
What happened to these messages? Don't we need to keep them?

Reply:
Oh, was the documentation here just outdated and these don't exist anymore?

Reply:
yes they are outdated